frontend/src/js/query-runner/DownloadResultsDropdownButton.tsx

```diff
@@ -43,56 +44,82 @@ const Separator = styled("div")`
 // might need to adjust this when adding more content.
 const dropdownOffset: [number, number] = [-37, 8]; // [skidding, distance] / default [0, 10]
```

Comment:
Now that the content in the dropdown is changed, this skidding / distance setting needs to change as well: the skidding helped align the right edge of the dropdown with the button element. Now with longer content, it's no longer aligned and sticks to the right edge of the screen.

To re-align things properly, here's a PR https://github.com/ingef/conquery/pull/2944